data/benefits/javascript/atmb-aide-covoiturage.yml

```
@@ -0,0 +1,316 @@
+      - "74054"
+      - "74079"
+      - "01081"
+      - "74014"
+      - "01109"
+      - "74226"
+      - "74072"
+      - "74179"
+      - "01269"
+      - "74090"
+      - "74178"
+      - "74252"
+      - "74304"
+      - "01298"
+      - "74069"
+      - "74118"
+      - "74024"
+      - "74081"
+      - "74243"
+      - "74078"
+      - "74206"
+      - "74166"
+      - "74002"
+      - "74123"
+      - "74108"
+      - "74262"
+      - "74160"
+      - "74193"
+      - "74020"
+      - "74187"
+      - "74144"
+      - "74235"
+      - "74043"
+      - "74272"
+      - "74143"
+      - "01153"
+      - "01104"
+      - "74198"
+      - "74159"
+      - "74105"
+      - "74021"
+      - "74065"
+      - "74038"
+      - "74291"
+      - "74254"
+      - "01143"
+      - "74302"
+      - "74048"
+      - "74051"
+      - "01397"
+      - "74145"
+      - "01210"
+      - "74265"
+      - "74059"
+      - "74261"
+      - "74084"
+      - "74197"
+      - "74016"
+      - "01152"
+      - "74013"
+      - "74006"
+      - "74176"
+      - "74011"
+      - "74177"
+      - "74003"
+      - "74204"
+      - "74122"
+      - "74086"
+      - "74025"
+      - "74222"
+      - "74066"
+      - "74067"
+      - "74221"
+      - "01407"
+      - "74253"
+      - "01436"
+      - "01257"
+      - "74074"
+      - "74215"
+      - "74294"
+      - "74186"
+      - "74260"
+      - "74110"
+      - "74133"
+      - "01189"
+      - "74103"
+      - "74135"
+      - "74007"
+      - "74101"
+      - "74040"
+      - "74223"
+      - "74199"
+      - "74209"
+      - "74241"
+      - "01071"
+      - "74005"
+      - "74128"
+      - "74189"
+      - "74094"
+      - "74194"
+      - "74147"
+      - "74140"
+      - "74219"
+      - "01114"
+      - "01274"
+      - "74071"
+      - "74042"
+      - "01135"
+      - "74130"
+      - "74244"
+      - "01247"
+      - "74264"
+      - "74087"
+      - "74010"
+      - "74309"
+      - "74126"
+      - "01401"
+      - "74019"
+      - "74275"
+      - "74107"
+      - "74049"
+      - "74267"
+      - "74305"
+      - "74044"
+      - "01209"
+      - "74082"
+      - "01281"
+      - "74307"
+      - "74027"
+      - "01103"
+      - "74184"
+      - "01413"
+      - "74268"
+      - "74053"
+      - "74180"
+      - "74315"
+      - "74301"
+      - "74112"
+      - "01360"
+      - "01448"
+      - "74031"
+      - "74314"
+      - "74018"
+      - "74080"
+      - "74276"
+      - "74162"
+      - "74284"
+      - "74174"
+      - "74269"
+      - "01288"
+      - "74293"
+      - "74263"
+      - "74167"
+      - "01357"
+      - "01078"
+      - "74045"
+      - "74093"
+  - type: age
+    operator: ">="
+    value: 18
+profils: []
+conditions:
+  - "Télécharger l'une des applications partenaires : Karos ou BlaBlaCar Daily."
+  - Effectuer un trajet dont la distance est comprise entre 4 et 80 kilomètres.
+  - Réaliser votre trajet en Haute-Savoie (a target="_blank" rel="noopener" title
```

```suggestion
  - Réaliser votre trajet en Haute-Savoie <a target="_blank" rel="noopener" title
```
Soucis de balise html